_tags

```diff
@@ -4,9 +4,9 @@ true: warn(@5@8@10@11@12@14@23-24@26@29@40), bin_annot, safe_string, debug
 <node_modules/**>: -traverse
 "formatTest": -traverse
 "src": include
-<src/*>: package(menhirLib unix compiler-libs.common ocamlbuild findlib easy-format str BetterErrors merlin_extend re.str)
-<src/reason_utop.*>: package(menhirLib utop str re.str)
+<src/*>: package(menhirLib unix compiler-libs.common ocamlbuild findlib easy-format BetterErrors merlin_extend re.str)
```

Comment:
Build is failing because Str got readded to pprint_ast - https://github.com/facebook/reason/blob/master/src/reason_pprint_ast.ml#L1443
Should be replaced w/ helper function here: https://github.com/facebook/reason/blob/master/src/reason_pprint_ast.ml#L1370